testing/trino-tests/src/test/java/io/trino/execution/TestCoordinatorDynamicFiltering.java

```diff
@@ -304,76 +289,52 @@ public void testRightJoinWithSelectiveBuildSide()
                         singleValue(BIGINT, 1L))));
     }
 
-    @Test(timeOut = 30_000)
-    public void testSemiJoinWithEmptyBuildSide()
+    @Test(timeOut = 30_000, dataProvider = "testJoinDistributionType")
+    public void testSemiJoinWithEmptyBuildSide(JoinDistributionType joinDistributionType, boolean coordinatorDynamicFiltersDistribution)
     {
         assertQueryDynamicFilters(
+                noJoinReordering(joinDistributionType, coordinatorDynamicFiltersDistribution),
                 "SELECT * FROM lineitem WHERE lineitem.suppkey IN (SELECT supplier.suppkey FROM tpch.tiny.supplier WHERE supplier.name = 'abc')",
                 TupleDomain.none());
     }
 
-    @Test(timeOut = 30_000)
-    public void testBroadcastSemiJoinWithEmptyBuildSide()
```

Comment:
why it's removed?

Reply:
it's redundant with the parameterized `testSemiJoinWithEmptyBuildSide` now